<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>split_dialog</class>
 <widget class="QDialog" name="split_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>绿信比编辑</string>
  </property>
  <widget class="QLabel" name="label_split">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>绿信比:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>模式:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_countdown">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Time">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Mode">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
   <item>
    <property name="text">
     <string>其他</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>无</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最小车辆响应</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最大车辆响应</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>行人响应</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最大车辆/行人响应</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>忽略相位</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_AsCoordPhase">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>作为协调相位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_AsFixedPhase">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>作为固定相位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_AsKeyPhase">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>作为关键相位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_phase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>相位:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_PhaseNumber">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>32</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>160</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
